### PR TITLE
QC-327 fix the path to the config file

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: Framework/src/runBasic.cxx [hunk 1]
```diff
@@ -133,6 +133,6 @@ std::string getConfigPath(const ConfigContext& config)
   // The the optional one by the user
   auto userConfigPath = config.options().get<std::string>("config-path");
   // Finally build the config path based on the default or the user-base one
-  std::string path = std::string("json:/") + (userConfigPath.empty() ? defaultConfigPath : userConfigPath);
```
Comment (knopers8) [inline Framework/src/runBasic.cxx:136]: That didn't work?

Comment (Barthelemy) [reply Framework/src/runBasic.cxx:136]: apparently it lacks a `/`. for sure on Mac it failed. I'll give it a try on CC7.

Comment (knopers8) [reply Framework/src/runBasic.cxx:136]: ok, for CC7 both worked as far as I remember

Comment (Barthelemy) [reply Framework/src/runBasic.cxx:136]: I cannot test at the moment due to the alicache problem.

Comment (Barthelemy) [reply Framework/src/runBasic.cxx:136]: ok it works both on Mac and CC7 now

Comment (knopers8) [reply Framework/src/runBasic.cxx:136]: ok, i don't mind either of these alternatives, just wanted to know if both work.

**Pull request merged**